tflite/python/tflitePythonVivante/.conf/createDockerComposeProduction.ps1

```diff
@@ -134,10 +146,10 @@ Set-Location -
 Write-Host -ForegroundColor DarkGreen "✅ Image rebuild and tagged"
 
 # push it
-Write-Host "Pushing it $dockerLogin/$($imageName):$tag ..."
+Write-Host "Pushing it $env:DOCKER_LOGIN/${imageName}:$tag ..."
 
-docker login --username $dockerLogin --password $psswd
-docker push $dockerLogin/$($imageName):$tag
+Write-Output "$psswd" | docker login $registry -u $dockerLogin --password-stdin
+docker push $env:DOCKER_LOGIN/${imageName}:$tag
 
 Write-Host -ForegroundColor DarkGreen "✅ Image push OK"
 
```

Comment:
I added a little fix on this file, on the  `$composeServices[$key].image.replace("`${DOCKER_LOGIN}", $dockerLogin)` line, to         $composeServices[$key].image.replace("`${DOCKER_LOGIN}", $env:DOCKER_LOGIN). Can you add that fix, please? You don't need to run the `try-update-template` task again, as it is a small change you can add it manually in this file.
[https://github.com/toradex/vscode-torizon-templates/commit/9195cdb9e84ecf7ec1cc3600f105017a573e08ae](https://github.com/toradex/vscode-torizon-templates/commit/9195cdb9e84ecf7ec1cc3600f105017a573e08ae)

Reply:
Surely, this is done.

If I misunderstood exactly what needed to be done, please let me know.